pkg/util/chmod/chmod.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package chmod
+
+import (
+	"os"
+)
+
+type Interface interface {
```

Comment:
I'm not super keen on the factoring of these.  I know it's supposed to be a trivially small util, yet it feels SO small.

I'm asking this, not telling.  would it be cleaner to define the Chmoder and Chowner interfaces at the consumer (kubelet) and define a merged util/fsutil/OwnershipManager struct with Chmod() and Chown() funcs?  Or is that even more artificial?